src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/Helpers.cs

```diff
@@ -24,7 +27,7 @@ internal static class Helpers
         /// We retrieve this character from triggerLocation.
         /// </param>
         /// <returns>Roslyn completion trigger</returns>
-        internal static RoslynTrigger GetRoslynTrigger(EditorAsyncCompletionData.CompletionTrigger trigger, SnapshotPoint triggerLocation)
+        public static RoslynTrigger GetRoslynTrigger(EditorAsyncCompletionData.CompletionTrigger trigger, SnapshotPoint triggerLocation)
```

Comment:
😕 Why are these changing? This file should have worked the same without any changes.

Reply:
I initially made some change in this file which was later deleted. And I changed accessibility to make them consistent with other place in the code while doing so (but did not revert it after)